bin/test-workflow/4_admin_cluster_prep.sh

```diff
@@ -3,27 +3,40 @@
 set -euo pipefail
 cd "$(dirname "$0")" || ( echo "cannot cd into dir" && exit 1 )
 
-PLATFORM="${PLATFORM:-kubernetes}"
 TIMEOUT="${TIMEOUT:-5m0s}"
 
 source utils.sh
 
 check_env_var CONJUR_APPLIANCE_URL
-check_env_var CONJUR_NAMESPACE
+check_env_var CONJUR_NAMESPACE_NAME
 check_env_var CONJUR_ACCOUNT
 check_env_var AUTHENTICATOR_ID
+if [[ "$CONJUR_OSS_HELM_INSTALLED" == "false" ]]; then
+  check_env_var CONJUR_FOLLOWER_URL
+fi
 
 set_namespace default
 
 # Prepare our cluster with conjur and authnK8s credentials in a golden configmap
 announce "Installing cluster prep chart"
 pushd ../../helm/conjur-config-cluster-prep > /dev/null
+  if [[ "$CONJUR_OSS_HELM_INSTALLED" == "true" ]]; then
     ./bin/get-conjur-cert.sh -v -i -s -u "$CONJUR_APPLIANCE_URL"
 
-    helm upgrade --install cluster-prep . -n "$CONJUR_NAMESPACE" --debug --wait --timeout $TIMEOUT \
+    helm upgrade --install cluster-prep . -n "$CONJUR_NAMESPACE_NAME" --debug --wait --timeout "$TIMEOUT" \
         --set conjur.account="$CONJUR_ACCOUNT" \
         --set conjur.applianceUrl="$CONJUR_APPLIANCE_URL" \
         --set conjur.certificateFilePath="files/conjur-cert.pem" \
         --set authnK8s.authenticatorID="$AUTHENTICATOR_ID"
-        
+  else
```

Comment:
Instead of two different branches of `get-conjur-cert` and `helm upgrade --install` could we create variables like `create_service_account` and `conjur_url` which are passed into a single invocation?

Reply:
I'm actually just implemented this for my next PR, as there would be a third branch, along with similar changes in `5_app_namespace_prep`. Could it stay there?

Reply:
Good catch, I like this idea.
Note that the `--set authnK8s.serviceAccount.name="conjur-cluster"` setting can be used in the single invocation (i.e. this name will work for both Conjur OSS and non-Conjur-OSS cases.